services/madoc-ts/src/routes/iiif/canvases/delete-canvas-summary.ts

```diff
@@ -0,0 +1,51 @@
+import { sql } from 'slonik';
+import { api } from '../../../gateway/api.server';
+import { RouteMiddleware } from '../../../types/route-middleware';
+import { userWithScope } from '../../../utility/user-with-scope';
+
+export const deleteCanvasSummary: RouteMiddleware<{ id: number }> = async context => {
+  const {siteId} = userWithScope(context, ['site.admin']);
+  const canvasId = context.params.id;
+
+  const siteApi = api.asUser({siteId});
+
+  // Fact checking stage.
+  const { site_count, manifest_count } = await context.connection.one(
+    // This will let us know if the manifest appears on any other sites.
+    // If === 1 then we can safely delete underlying resource.
+    // TODO can a canvas appear in multiple manifests on the same site?
+    sql<{ site_count: number; manifest_count: number }>`
+      select COUNT(distinct site_id) as site_count, COUNT(*) as manifest_count from iiif_derived_resource where resource_id=${canvasId}
```

Comment:
`canvas_count` in this file should probably be `canvas_count`

Reply:
Do you mean `manifest_count` should be `canvas_count` instead? What I was trying to do here was work out how many manifests this canvas appears in

Reply:
Ah right, yes that makes sense 👍